src/js/helpers/local_storage.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+var default_cookies = require('./cookies');
+var local_storage_session_length = 0;
+
+module.exports = Object.assign({}, default_cookies, {
+
+  set: function(name, value, minutes, domain, excl_subdomains) {
+    // Don't break the build
+    domain = '';
+    excl_subdomains = false;
+
+    // `item` is an object which contains the original value
+    // as well as the time when it's supposed to expire
+    var item = {
+      value: this.encodeData( value ),
+      expiry: (new Date()).getTime() + ( Math.max( minutes, local_storage_session_length ) * 60 * 1000 ),
```

Comment:
> If minutes is falsy (including 0), it will use local_storage_session_length.

We could also do an explicit `minutes === undefined` check, as sometimes `0` has a convention to denote no limit.